admin-ui/app/components/bootstrap/bootstrap.html

```diff
@@ -7,8 +7,8 @@
         <label>
           <input type="checkbox" ng-model="bootstrap.types.android">
               <span class="ups-has-icon ups-variant-android">
-                <strong>Android</strong><br>
-                using Firebase Cloud Messaging
+                <strong translate>Android</strong><br>
```

Comment:
Yes, it will be, and not sure. just marked it that way for languages may wants to be translated. I don't even know if there are any but I'm not pretty sure if it is 100% not necessary.